test/external/external_test_onnx_backend.py

```diff
@@ -139,20 +145,26 @@ def supports_device(cls, device: str) -> bool:
 backend_test.exclude('test_resize_upsample_sizes_cubic_*') # unsure how to implement cubic
 
 # rest of the failing tests
-backend_test.exclude('test_averagepool_2d_dilations_cpu') # dilations != 1 not supported for avgpool
+backend_test.exclude('test_averagepool_2d_dilations_*') # dilations != 1 not supported for avgpool in tensor.py
```

Comment:
You should just be able to add dilation to `avg_pool2d` in `tensor.py` just like `max_pool2d`